Dockerfile

```diff
@@ -1,17 +1,24 @@
+# build stage
 FROM docker.io/node:18-bullseye as build-stage
 
 RUN mkdir /app && chown -R node:node /app
+
 WORKDIR /app
 
-COPY package.json .
-COPY package-lock.json .
+COPY package.json package-lock.json ./
 RUN npm ci
+
 COPY . ./
```

Comment:
Change it please - copy only the files you really need - otherwise you'll copy everything, including some random files that you don't really need (and shouldn't have) in the build stage.